### PR TITLE
Ensure ip forward for VLAN

### PR DESCRIPTION
**Pull request by yaocw2020**
issue: https://github.com/harvester/harvester/issues/841

### CODE REVIEW

File: pkg/network/iface/link.go [hunk 1]
```diff
@@ -203,6 +208,63 @@ func (l *Link) unsetRules4DHCP() error {
...
+	}
+
+	for _, rule := range rules {
+		if strings.HasPrefix(rule, "-P "+chainForward) {
+			if strings.Fields(rule)[2] == "ACCEPT" {
+				return nil
+			}
+			break
+		}
+	}
```
Comment (gitlawr) [inline pkg/network/iface/link.go:239]: Use `ipt.Exists()` for the check

Comment (yaocw2020) [reply pkg/network/iface/link.go:239]: What we check is the policy, not a rule.  The function `ipt.Exists` doesn't work in this case.

**Review by gitlawr — APPROVED** [review]
LGTM

**Review by guangbochen — APPROVED** [review]
LGTM

**Pull request merged**